### PR TITLE
Update to core 2.0.1

### PR DESCRIPTION
**Pull request by tgoyne**

### CODE REVIEW

File: dependencies.list [hunk 1]
```diff
@@ -1,4 +1,4 @@
 VERSION=2.0.0
-REALM_CORE_VERSION=2.0.0
+REALM_CORE_VERSION=2.0.1
```
Comment (bdash) [inline dependencies.list]: Hmm, I'm not sure this is even _used_ at this point. It looks like it's only used by `build.sh download-core`, and we've replaced all calls to this with `build.sh download-sync`.

Comment (jpsim) [inline dependencies.list]: Correct.

**Comment by tgoyne** [comment]
This now actually uses core 2.0.1. I think all of the failing jobs on CI are spurious.

**Comment by tgoyne** [comment]
All of the failing jobs work for me locally, so just going to blame CI issues for them.

**Pull request merged**